compiler/z/codegen/S390Instruction.hpp

```diff
@@ -2551,6 +2551,7 @@ class S390RSInstruction : public TR::S390RegInstruction
    int16_t                 _sourceImmediate;
    int8_t                  _maskImmediate;
    bool                    _hasMask;
+   bool                    _hasImmediate;
```

Comment:
Would it be possible to add Doxygen documentation for the newly added field and API?